llvm/lib/Target/CheerpBackend/CheerpTargetTransformInfo.cpp

```diff
@@ -0,0 +1,127 @@
+#include "CheerpTargetTransformInfo.h"
+#include "llvm/CodeGen/CostTable.h"
+#include "llvm/Support/Debug.h"
+using namespace llvm;
+
+#define DEBUG_TYPE "wasmtti"
+
+TargetTransformInfo::PopcntSupportKind
```

Comment:
I would remove this function and add only stuff that is needed for simd, to avoid introducing unwanted changes